css/welcome.css

```diff
@@ -0,0 +1,258 @@
+/* welcome page styling goes here */
```

Comment:
idk if it's Netlify's deploy preview being weird since when I shrink my browser your changes look great, but when I open it on my phone it looks like this:
![IMG_2914](https://user-images.githubusercontent.com/47371080/206876443-86d2d230-8081-4b30-9a98-1ad30a989c9f.jpg)

Can you double check on your end?

Reply:
hmm odd... I'm opening the page from my mobile brave browser and it seems fine... even after tweaking display settings. (using a Pixel 4a)

what kind of phone are you using?

Reply:
iphone 8. It's probably just me if you don't see it on your phone